go/pkg/cas/upload_test.go

```diff
@@ -453,6 +470,185 @@ func TestStreaming(t *testing.T) {
 	}
 }
 
```

Comment:
I think there's some value on adding a test that checks the same pathspec with two different `Allowlist` don't interfere with each other in a same `Upload` call.

Reply:
Dne